dbt/adapters/athena/__version__.py

```diff
@@ -1 +1 @@
-version = "1.5.1"
+version = "1.5.2"
```

Comment:
could you please revert this? we only maintainers bump the version manually before releasing, thanks

Reply:
Make sense! Just reverted the version change.